### PR TITLE
Refactor and redesing google and facebook login

### PR DESCRIPTION
**Pull request by LesiaM-T**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

Redesing Modal window for LogIn

## Summary of change

Created a component for facebook login, took it out from container. Changed styles for buttons for Google and Facebook login.

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/login/google-login/google-login.jsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
 ﻿import React from "react";
 import { GoogleLogin as Login } from "react-google-login";
 import PropTypes from "prop-types";
+import "../google-facebook-login.scss";
```
Comment (VadymKharyton) [inline src/components/login/google-login/google-login.jsx]: Move this import to the bottom of other imports.
All CSS style imports need to placed after all modules imports.

File: src/components/login/google-login/google-login.jsx [hunk 2]
```diff
@@ -14,12 +19,12 @@ const GoogleLogin = ({
...
           <button
             type="button"
-            className="btnGoogle"
+            className="btn-google"
             onClick={renderProps.onClick}
             disabled={renderProps.disabled}
           >
-            <i className="fab fa-google blue fa-lg" />
-            <span>Log in</span>
+            <FcGoogle className="icon" />
```
Comment (VadymKharyton) [inline src/components/login/google-login/google-login.jsx]: Please rename this className according to BEM methodology, it should not be just `icon`. If we add more icons, there will be className conflict

File: src/components/login/google-login/google-login.jsx [hunk 3]
```diff
@@ -14,12 +19,12 @@ const GoogleLogin = ({
...
+            <span className="text">{GOOGLE_LOGIN}</span>
```
Comment (VadymKharyton) [inline src/components/login/google-login/google-login.jsx]: Same for `text` className

File: src/components/login/facebook-login/facebook-login.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FacebookLogin from "react-facebook-login";
+import "../google-facebook-login.scss";
+import { FaFacebook } from "react-icons/fa";
+import ButtonNames from "../../../constants/ButtonNamesConstants";
+
+const { FACEBOOK_LOGIN } = ButtonNames;
+
+const LoginFacebook = ({ config }, props) => {
```
Comment (VadymKharyton) [inline src/components/login/facebook-login/facebook-login.jsx]: Why we have two separate parameters here, there should be only one instead. Can you please verify if everything is correct here

File: src/components/login/facebook-login/facebook-login.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+import FacebookLogin from "react-facebook-login";
+import "../google-facebook-login.scss";
```
Comment (VadymKharyton) [inline src/components/login/facebook-login/facebook-login.jsx]: Move this import to the bottom of other imports.
All CSS style imports need to placed after all modules imports.

File: src/components/login/facebook-login/facebook-login.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import { FaFacebook } from "react-icons/fa";
+import ButtonNames from "../../../constants/ButtonNamesConstants";
+
+const { FACEBOOK_LOGIN } = ButtonNames;
+
+const LoginFacebook = ({ config }, props) => {
+  const responseFacebook = response => {
+    if (!response.email) {
+      props.setErrorAlert("Please add email to your facebook account!");
```
Comment (VadymKharyton) [inline src/components/login/facebook-login/facebook-login.jsx]: Can all props be destructed inside function params

File: src/components/login/google-login/google-login.jsx (hunk 1 above)
Comment (LesiaM-T) [reply src/components/login/google-login/google-login.jsx]: Okay, thank you, moved.

File: src/components/login/google-login/google-login.jsx (hunk 2 above)
Comment (LesiaM-T) [reply src/components/login/google-login/google-login.jsx]: Ok, changed.

File: src/components/login/google-login/google-login.jsx (hunk 3 above)
Comment (LesiaM-T) [reply src/components/login/google-login/google-login.jsx]: Ok, changed.

File: src/components/login/facebook-login/facebook-login.jsx (hunk 4 above)
Comment (LesiaM-T) [reply src/components/login/facebook-login/facebook-login.jsx]: Changed

File: src/components/login/facebook-login/facebook-login.jsx (hunk 5 above)
Comment (LesiaM-T) [reply src/components/login/facebook-login/facebook-login.jsx]: Okay, thank you, moved.

File: src/components/login/facebook-login/facebook-login.jsx (hunk 6 above)
Comment (LesiaM-T) [reply src/components/login/facebook-login/facebook-login.jsx]: Yes. Changed.

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**